src/ontology/cl-edit.owl

```diff
@@ -7798,15 +7800,15 @@ SubClassOf(obo:CL_0000507 obo:CL_0000167)
 
 # Class: obo:CL_0000508 (type G enteroendocrine cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "ISBN:0517223651") Annotation(oboInOwl:hasDbXref "MESH:D019863") obo:IAO_0000115 obo:CL_0000508 "An endocrine cell found in the pyloric gland mucosa (antral mucosa) of the stomach of mammals and responsible for the secretion of gastrin and enkephalin. Most abundant in pyloric antrum, pyramidal in form with a narrow apex bearing long microvilli.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "ISBN:0517223651") Annotation(oboInOwl:hasDbXref "MESH:D019863") obo:IAO_0000115 obo:CL_0000508 "An endocrine cell found in the stomach, duodenum and pancreas and is responsible for the secretion of gastrin and enkephalin. Most abundant in pyloric antrum, pyramidal in form with a narrow apex bearing long microvilli.")
```

Comment:
can you please add references to support the new textual defs? I remember when we discussed it in the meeting, there wasn't enough evidence to show that the cell is in the pancreas but we found few references showing their presence in the stomach and duodenum.

Reply:
new defs added